test/second/main.tf

```diff
@@ -0,0 +1,12 @@
+module "public" {
+  source = "git@github.com:patrickjahns/terraform-module-public.git?ref=0.0.0"
+}
+
+module "private" {
```

Comment:
I presume this test probably wouldn't work for us

Reply:
I honestly don't know... just trying to get it to work on the wrangler-watchmaker project... https://github.com/plus3it/wrangler-watchmaker/runs/687907941?check_suite_focus=true

Saw there were upstream updates having to do with the directory, so figured it was worth updating from upstream.